monorail/tests/test_settings.py

```diff
@@ -3,23 +3,28 @@
 
 from monorail.settings import *
 from monorail.scenarios import *
+from monorail.koon.input import UserInput
+import pygame
 
 
 class TestSettings:
 
     def test_init( self ):
-        game_data = GameData()
+        pygame.init()
+        game_data = GameData(UserInput())
 
     def test_is_single_player( self ):
-        game_data = GameData()
+        pygame.init()
+        game_data = GameData(UserInput())
 
         assert game_data.is_single_player() == True or \
                game_data.is_single_player() == False
 
     def test_get_quest( self ):
-        game_data = GameData()
+        pygame.init()
```

Comment:
Original code tests specific methods of `GameData` class. It is not [integration](https://en.wikipedia.org/wiki/Integration_testing) but [unit testing](https://en.wikipedia.org/wiki/Unit_testing). What you did is more like integration testing, but even so you have missed clean up code: `pygame.quit()`.

I suppose integration testing have to be in separate module.